ios/App/App/plugins/AbsAudioPlayer.swift

```diff
@@ -35,15 +34,14 @@ public class AbsAudioPlayer: CAPPlugin {
     }
     
     func restorePlaybackSession() async {
-        // We don't need to restore if we have an active session
-        guard PlayerHandler.getPlaybackSession() == nil else { return }
-        
         do {
             // Fetch the most recent active session
             let activeSession = try await Realm().objects(PlaybackSession.self).where({
                 $0.isActiveSession == true && $0.serverConnectionConfigId == Store.serverConfig?.id
             }).last?.freeze()
+            
             if let activeSession = activeSession {
+                PlayerHandler.stopPlayback(currentSessionId: activeSession.id)
```

Comment:
When reloading the session, we should stop playback so audio doesn't play while the next line's network connection is made.

This is an issue because the Vue code will change playback rate during startup, which will unpause the audio. By stopping playback, changing the playback rate ensures audio doesn't start playing.